src/tools/wasm-ctor-eval.cpp

```diff
@@ -369,7 +369,10 @@ int main(int argc, const char* argv[]) {
       .add("--ctors", "-c", "Comma-separated list of global constructor functions to evaluate",
            Options::Arguments::One,
            [&](Options* o, const std::string& argument) {
-             ctorsString = argument;
+               if (!argument.empty() && argument[0] == '@')
```

Comment:
indentation needs some fixing here (2 spaces), and also need `{`, `}` around the if and body.

a larger issue is that we should probably make a utility method in `src/support/file.h` for expanding a response file. perhaps we should even do this by default? or have an option on Options that they have this feature or not?